server/sonar-server/src/test/java/org/sonar/server/measure/index/ProjectMeasuresIndexerTest.java

```diff
@@ -131,6 +132,7 @@ public void index_one_project() throws Exception {
     assertThat(esTester.getIds(INDEX_TYPE_PROJECT_MEASURES)).containsOnly(project.uuid());
   }
 
+  @Ignore
   @Test
   public void update_existing_document_when_indexing_one_project() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)